diff -urN credcheck-3.0/Makefile credcheck-3.0_patched/Makefile
--- credcheck-3.0/Makefile	2024-12-31 17:30:25.000000000 +0100
+++ credcheck-3.0_patched/Makefile	2025-04-09 10:17:56.012903375 +0200
@@ -4,8 +4,8 @@
 
 # Uncomment the following two lines to enable cracklib support, adapt the path
 # to the cracklib dictionary following your distribution
-#PG_CPPFLAGS = -DUSE_CRACKLIB '-DCRACKLIB_DICTPATH="/usr/lib/cracklib_dict"'
-#SHLIB_LINK = -lcrack
+PG_CPPFLAGS = -DUSE_CRACKLIB '-DCRACKLIB_DICTPATH="/usr/share/cracklib/pw_dict"'
+SHLIB_LINK = -lcrack
 
 PG_CPPFLAGS += -Wno-ignored-attributes
 
